Source/Initialization/WarpXInitData.cpp

```diff
@@ -119,9 +119,6 @@ WarpX::InitData ()
     else
     {
         InitFromCheckpoint();
-        if (is_synchronized) {
-            ComputeDt();
-        }
```

Comment:
@RemiLehe Do you happen to remember why this was removed? I thought `is_synchronized` would guard whether or not we need to re-compute the time step, but I think I might be missing something.

Reply:
Hm, I don't remember exactly. 
It may be that the time step is read directly from the checkpoint file during `InitFromCheckpoint`, and therefore we do not need to recompute it (whether or not `is_synchronized` is true).

Reply:
Yes, I also think it was something along these lines. I think `dt` is indeed read in `InitFromCheckpoint`, here: https://github.com/ECP-WarpX/WarpX/blob/7abef41c4e37e78dabeeb800332998cb0c46cdd5/Source/Diagnostics/WarpXIO.cpp#L117-L124